samples/src/main/scala/SparkStreamingKustoSink.scala

```diff
@@ -9,6 +9,10 @@ import org.apache.spark.sql.functions._
 /** ************************************************/
 /*          STREAMING SINK EXAMPLE                */
 /** ************************************************/
+
+// To enable faster ingestion into kusto, set a  minimal value for the batching ingestion policy:
```

Comment:
This should be followed with a warning for pricing and maybe even a link to the docs : https://kusto.azurewebsites.net/docs/concepts/batchingpolicy.html?q=batching

Reply:
Would be better to point to the external documentation at https://docs.microsoft.com/en-us/azure/kusto/concepts/ 
The only problem is that the batching policy section was not published. Vladik made the necessary change already, I just don't see it reflected on the documentation site yet